client/src/app/pages/not-found/main/not-found-main.component.html

```diff
@@ -1,3 +1,4 @@
+<nb-layout style="display:none"></nb-layout>
```

Comment:
I remember Nebular requiring `nb-layout` to have its components working, is that what this is doing?

Reply:
```nb-layout``` component hides spinner on initialization, so that's why I'm using it.